graspologic/utils/utils.py

```diff
@@ -299,50 +316,95 @@ def to_laplace(graph, form="DAD", regularizer=None):
            [0.70710678, 0.        , 0.        ]])
 
     """
+
```

Comment:
`_sparse` and `_dense` have a lot of repeated code, maybe rethink what needs to be refactored, or whether should still be one function, etc.